pkg/kubelet/cm/container_manager.go

```diff
@@ -44,6 +46,9 @@ type ContainerManager interface {
 
 	// GetQOSContainersInfo returns the names of top level QoS containers
 	GetQOSContainersInfo() QOSContainersInfo
+
+	// UpdateQOSCgroups returns the names of top level QoS containers
```

Comment:
godoc is wrong, it performs housekeeping updates to ensure that the top level QoS containers have their desired state.  also note that this operation is thread-safe.